git_template/hooks/commit-msg

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+local_hook="$HOME"/.git_template.local/hooks/commit-msg
+
+if [ -f "$local_hook" ]; then
+  . "$local_hook"
+fi
+
```

Comment:
Let's remove this whitespace here.

Reply:
Good catch. Fixed in https://github.com/thoughtbot/dotfiles/pull/649/commits/eeb52a87f6d869d8790936f8dd628c08744d0b5d